### PR TITLE
feat: expose fair about, links, tickets_link fields

### PR DESCRIPTION
**Pull request by javamonn**
Implements [LD-99](https://artsyproduct.atlassian.net/browse/LD-99)

Exposes fair `about`, `links`, and `tickets_link` fields.

### CODE REVIEW

File: src/schema/fair.ts [hunk 1]
```diff
@@ -230,6 +236,9 @@ const FairType = new GraphQLObjectType({
     tagline: {
       type: GraphQLString,
     },
+    tickets_link: {
```
Comment (javamonn) [inline src/schema/fair.ts]: On a passthrough field like this, do we want to camelCase?

Comment (alloy) [reply src/schema/fair.ts]: Yup, every new field should be camelCase.

Comment (damassi) [reply src/schema/fair.ts]: @alloy - there's no way to rename a field with attaching a resolver, right?

Comment (orta) [reply src/schema/fair.ts]: Nope, gotta do the `({ snake_case }) => snake_case` dance I'm afraid

**Pull request merged**